### PR TITLE
#50 Request meeting

### PR DESCRIPTION
**Pull request by vmtikk**

### CODE REVIEW

File: WebAPI/Controllers/WebComponentsController.cs
Comment (vijupa) [inline WebAPI/Controllers/WebComponentsController.cs:68]: #--MEETING_YEAR--#, #--MEETING-SEQUENCE_NUM--# could be more descriptive

File: WebAPI/StorageClient/StorageConnection.cs
Comment (jmsalonen) [inline WebAPI/StorageClient/StorageConnection.cs:20]: Shouldn't this be STORAGE_URL ?

File: WebAPI/Controllers/WebComponentsController.cs
Comment (vmtikk) [reply WebAPI/Controllers/WebComponentsController.cs]: fixed

File: WebAPI/StorageClient/StorageConnection.cs
Comment (vmtikk) [reply WebAPI/StorageClient/StorageConnection.cs]: fixed

**Review by vijupa — APPROVED** [review]
For me this is ok, but since it contains UI changes, lets wait for @koskelav approval before merging..

**Review by jmsalonen — APPROVED** [review]

**Pull request merged**
